company/okrs/2020_q1.md

```diff
@@ -82,13 +82,13 @@
    1. At least four customers are running structural searches and metrics to validate this KR are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
    1. At least four customers are running AND/OR/NOT queries and metrics to validate this KR are tracked in [pings](https://docs.sourcegraph.com/admin/pings).
 
-### Automation
+### Campaigns
 
 _This OKR is a joint effort between [web](#web) and [core services](#core-servies)._
 
-1. **Automation is polished and adopted by customers**
-   1. Document Automation and write a blog post about it.
-   1. Automation is enabled by default and not behind a feature flag.
+1. **Campaigns is polished and adopted by customers**
+   1. Document campaigns and write a blog post about it.
+   1. Campaigns is enabled by default and not behind a feature flag.
```

Comment:
```suggestion
   1. Campaigns are enabled by default and not behind a feature flag.
```

Reply:
This was intentional, but I'm fine either way.

Reply:
I think it would be helpful to add guidance like this to the documentation styleguide. Do we respect plurality rules? Or because we are treating it as a noun it is different? 

Alternatively "Code change management is..."